[BUG] ercole crash one day after the start

**Describe the bug**
ercole crash one day after the start

**Expected behavior**
Ercole is still running after one day

**To Reproduce**
1. `./ercole server`
2. `./ercole fire-hostdata fixture/test_dataservice_hostdata_v1_*.json`
3. wait one day

**Screenshots, logs, stdout, stderr, asciinema**
```
amreo@amreo-ubuntu-pc:~/src/gopath/src/github.com/ercole-io/ercole$ go run main.go -c ~/.config/ercole.toml serve
[20-09-01 17:05:41][SERV][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/cmd/serve.go:109] Migrating...                                      
[20-09-01 17:05:41][DATA][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/data-service/database/database.go:68] MongoDatabase is connected to MongoDB! mongodb://localhost:27017/ercole
[JobRunner] 2020/09/01 - 17:05:41 Started...  
[20-09-01 17:05:41][DATA][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/cmd/serve.go:181] Start data-service: listening at 11111            
[20-09-01 17:05:41][ALRT][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/alert-service/database/database.go:66] MongoDatabase is connected to MongoDB! mongodb://localhost:27017/ercole
[JobRunner] 2020/09/01 - 17:05:41 Started...  
[20-09-01 17:05:41][ALRT][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/alert-service/service/service.go:81] Start alert-service/queue                         
[20-09-01 17:05:41][ALRT][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/cmd/serve.go:236] Start alert-service: listening at 11112           
[20-09-01 17:05:41][APIS][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/api-service/database/database.go:173] MongoDatabase is connected to MongoDB! mongodb://localhost:27017/ercole
[20-09-01 17:05:41][APIS][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/cmd/serve.go:291] Start api-service: listening at 11113             
[20-09-01 17:05:41][CHRT][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/chart-service/database/database.go:64] MongoDatabase is connected to MongoDB! mongodb://localhost:27017/ercole
[20-09-01 17:05:41][CHRT][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/cmd/serve.go:346] Start chart-service: listening at 11116           
[20-09-01 17:05:41][REPO][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/repo-service/service/sftp.go:64] Start repo-service/sftp: listening at 11115       
[20-09-01 17:05:41][REPO][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/repo-service/service/http.go:55] Start repo-service/http: listening at 11114       
[20-09-01 17:05:42][APIS][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/utils/loggingHandler.go:31] HTTP Request                                      endpoint=GET /agreements/oracle/database?agreement-id=&available-count-gte=-1&available-count-lte=-1&catch-all=&csi=&item-description=&licenses-count-gte=-1&licenses-count-lte=-1&metrics=all&part-id=&search=&unlimited=&users-count-gte=-1&users-count-lte=-1 statusCode=200 accept= clientSocket=127.0.0.1:41140 serverSocket=127.0.0.1:11113 size=3 timeStamp=2020-09-01 17:05:42.751042614 +0200 CEST m=+1.728955528 userAgent=Go-http-client/1.1 
[20-09-01 17:28:46][APIS][INFO][/home/amreo/src/gopath/src/github.com/ercole-io/ercole/utils/loggingHandler.go:31] HTTP Request                                      endpoint=GET /agreements/oracle/database?agreement-id=&available-count-gte=-1&available-count-lte=-1&catch-all=&csi=&item-description=&licenses-count-gte=-1&licenses-count-lte=-1&metrics=all&part-id=&search=&unlimited=&users-count-gte=-1&users-count-lte=-1 statusCode=200 accept= clientSocket=127.0.0.1:42810 serverSocket=127.0.0.1:11113 size=3 timeStamp=2020-09-01 17:28:46.18306467 +0200 CEST m=+1385.160977582 userAgent=Go-http-client/1.1 
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
goroutine 11853 [running]:
runtime/debug.Stack(0xeb47c0, 0xc00042c370, 0x15f1e00)
	/usr/lib/go-1.15/src/runtime/debug/stack.go:24 +0x9f
github.com/bamzi/jobrunner.(*Job).Run.func1()
	/home/amreo/src/gopath/pkg/mod/github.com/bamzi/jobrunner@v0.0.0-20190810144113-852b7ca4d475/jobrunner.go:55 +0xd4
panic(0xe29260, 0x15f1e00)
	/usr/lib/go-1.15/src/runtime/panic.go:969 +0x175
github.com/sirupsen/logrus.(*Logger).level(...)
	/home/amreo/src/gopath/pkg/mod/github.com/sirupsen/logrus@v1.5.0/logger.go:300
github.com/sirupsen/logrus.(*Logger).IsLevelEnabled(...)
	/home/amreo/src/gopath/pkg/mod/github.com/sirupsen/logrus@v1.5.0/logger.go:322
github.com/sirupsen/logrus.(*Logger).Logf(0x0, 0x4, 0xf5310c, 0x4c, 0xc000443eb8, 0x2, 0x2)
	/home/amreo/src/gopath/pkg/mod/github.com/sirupsen/logrus@v1.5.0/logger.go:144 +0x26
github.com/sirupsen/logrus.(*Logger).Infof(...)
	/home/amreo/src/gopath/pkg/mod/github.com/sirupsen/logrus@v1.5.0/logger.go:160
github.com/ercole-io/ercole/data-service/service.(*CurrentHostCleaningJob).Run(0xc000157400)
	/home/amreo/src/gopath/src/github.com/ercole-io/ercole/data-service/service/current_host_cleaning_job.go:59 +0x20f
github.com/bamzi/jobrunner.(*Job).Run(0xc0001ac140)
	/home/amreo/src/gopath/pkg/mod/github.com/bamzi/jobrunner@v0.0.0-20190810144113-852b7ca4d475/jobrunner.go:75 +0x133
gopkg.in/robfig/cron%2ev3.(*Cron).startJob.func1(0xc0003c7a40, 0x11159e0, 0xc0001ac140)
	/home/amreo/src/gopath/pkg/mod/github.com/robfig/cron/v3@v3.0.0/cron.go:307 +0x65
created by gopkg.in/robfig/cron%2ev3.(*Cron).startJob
	/home/amreo/src/gopath/pkg/mod/github.com/robfig/cron/v3@v3.0.0/cron.go:305 +0x73

[signal SIGSEGV: segmentation violation code=0x1 addr=0x2c pc=0x754ec6]

goroutine 11853 [running]:
log.(*Logger).Panic(0xc00042c370, 0xc000443bd0, 0x3, 0x3)
	/usr/lib/go-1.15/src/log/log.go:221 +0xac
github.com/bamzi/jobrunner.(*Job).Run.func1()
	/home/amreo/src/gopath/pkg/mod/github.com/bamzi/jobrunner@v0.0.0-20190810144113-852b7ca4d475/jobrunner.go:55 +0x18a
panic(0xe29260, 0x15f1e00)
	/usr/lib/go-1.15/src/runtime/panic.go:969 +0x175
github.com/sirupsen/logrus.(*Logger).level(...)
	/home/amreo/src/gopath/pkg/mod/github.com/sirupsen/logrus@v1.5.0/logger.go:300
github.com/sirupsen/logrus.(*Logger).IsLevelEnabled(...)
	/home/amreo/src/gopath/pkg/mod/github.com/sirupsen/logrus@v1.5.0/logger.go:322
github.com/sirupsen/logrus.(*Logger).Logf(0x0, 0x4, 0xf5310c, 0x4c, 0xc000443eb8, 0x2, 0x2)
	/home/amreo/src/gopath/pkg/mod/github.com/sirupsen/logrus@v1.5.0/logger.go:144 +0x26
github.com/sirupsen/logrus.(*Logger).Infof(...)
	/home/amreo/src/gopath/pkg/mod/github.com/sirupsen/logrus@v1.5.0/logger.go:160
github.com/ercole-io/ercole/data-service/service.(*CurrentHostCleaningJob).Run(0xc000157400)
	/home/amreo/src/gopath/src/github.com/ercole-io/ercole/data-service/service/current_host_cleaning_job.go:59 +0x20f
github.com/bamzi/jobrunner.(*Job).Run(0xc0001ac140)
	/home/amreo/src/gopath/pkg/mod/github.com/bamzi/jobrunner@v0.0.0-20190810144113-852b7ca4d475/jobrunner.go:75 +0x133
gopkg.in/robfig/cron%2ev3.(*Cron).startJob.func1(0xc0003c7a40, 0x11159e0, 0xc0001ac140)
	/home/amreo/src/gopath/pkg/mod/github.com/robfig/cron/v3@v3.0.0/cron.go:307 +0x65
created by gopkg.in/robfig/cron%2ev3.(*Cron).startJob
	/home/amreo/src/gopath/pkg/mod/github.com/robfig/cron/v3@v3.0.0/cron.go:305 +0x73
exit status 2
```

**Desktop (please complete the following information):**
 - OS: 18.04.5 LTS (Bionic Beaver)
 - ercole version: 306995f7dd456240e4dddd95435921dec38a44b7 
 - the configuration is the default (https://github.com/ercole-io/ercole/blob/231d3816b5b10a7d27e7e24ee677d2b633b2d8ec/config.toml).

**Additional context (optional)**
I suspect it's related to the jobs.

**Possible solution (optional)**
A possibile workaround is to disable the jobs.

--- a/data-service/service/service.go
+++ b/data-service/service/service.go
@@ -63,12 +63,12 @@ func (hds *HostDataService) Init() {
 	//Start cron jobs
 	jobrunner.Start()
 
-	currentHostCleaningJob := &CurrentHostCleaningJob{hostDataService: hds, TimeNow: hds.TimeNow, Database: hds.Database, Config: hds.Config}
+	currentHostCleaningJob := &CurrentHostCleaningJob{hostDataService: hds, TimeNow: hds.TimeNow, Database: hds.Database, Config: hds.Config, Log: hds.Log}
 	if err := jobrunner.Schedule(hds.Config.DataService.CurrentHostCleaningJob.Crontab, currentHostCleaningJob); err != nil {
 		hds.Log.Errorf("Something went wrong scheduling CurrentHostCleaningJob: %v", err)
 	}
 
-	archivedHostCleaningJob := &ArchivedHostCleaningJob{hostDataService: hds, TimeNow: hds.TimeNow, Database: hds.Database, Config: hds.Config}
+	archivedHostCleaningJob := &ArchivedHostCleaningJob{hostDataService: hds, TimeNow: hds.TimeNow, Database: hds.Database, Config: hds.Config, Log: hds.Log}
 	if err := jobrunner.Schedule(hds.Config.DataService.ArchivedHostCleaningJob.Crontab, archivedHostCleaningJob); err != nil {
 		hds.Log.Errorf("Something went wrong scheduling ArchivedHostCleaningJob: %v", err)
 	}

